instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-resteasy-3.0/jaxrs-2.0-resteasy-3.0.gradle

```
@@ -4,7 +4,8 @@ muzzle {
   // Cant assert fails because muzzle assumes all instrumentations will fail
   // Instrumentations in jaxrs-2.0 will pass
 
-  // Resteasy changes a class's package in 3.1.0 then moves it back in 3.5.0 
+  // Resteasy changes a class's package in 3.1.0
+  // 3.0 is continued in 3.5, up to 4; 3.1 is continued in 4.0
```

oh wow I hadn't noticed this, suggestion to clarify further:
```suggestion
  // Resteasy changes a class's package in 3.1.0 then moves it back in 3.5.0 and then moves it forward again in 4.0.0
  // so the jaxrs-2.0-resteasy-3.0 module applies to [3.0, 3.1) and [3.5, 4.0)
  // and the jaxrs-2.0-resteasy-3.1 module applies to [3.5, 4.0)
```